front-end/src/components/Table/PendingOrderTable/helperComponents.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import clsx from "clsx";
+
+import {
+  TableCell,
+  Typography,
+  Paper,
+  TablePagination,
+  Button,
+} from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+import { AssignmentRounded as AssignmentRoundedIcon } from "@material-ui/icons";
+
+const StyledTableCell = withStyles((theme) => ({
+  head: {
+    backgroundColor: theme.palette.paperBackground.sub,
+    color: "white",
+  },
+}))(TableCell);
+
+export const chooseTableCellHeader = (indexColumnName, state) => {
+  const { names } = state;
+  const type = names[indexColumnName];
+
+  return (
+    <StyledTableCell key={indexColumnName} align={'center'}>{type}</StyledTableCell>
+  );
+};
+
+export const chooseTableCell = (type, isLastRow, classes, order) => {
+  return (
+    <TableCell
+      align="center"
+      className={clsx(classes.tableCell, {
+        [classes.lastRow]: isLastRow(),
+        [classes.lastLeftCell]: isLastRow() && type === "Type",
+        [classes.lastRightCell]:
+        isLastRow() && type === "Actions",
+      })}
+    >
+      <div>
+        <Typography noWrap>
+          {chooseTableCellValue(type, order, classes)}
+        </Typography>
+      </div>
+    </TableCell>
+  );
+};
+
+export const chooseTableCellValue = (type, order, classes) => {
+  const {
+    isTypeBuy,
+    companyCode,
+    quantity,
+    option,
+    limitPrice,
+    brokerage,
+    tradeValue,
```

Comment:
Currently there is no `tradeValue` attribute in `order` object. Use this formula `tradeValue = limitPrice * quantity + brokerage` to calculate tradeValue.